js/interface.js

```diff
@@ -40,7 +40,7 @@ var defaultForm = [
   '</div>',
   '<hr class="hr-normal">',
   '<div class="form-btns clearfix">',
-    '<button type="submit" class="btn btn-primary pull-right">Submit</button>',
+    '<button type="submit" class="btn btn-primary pull-right" data-edit-label="Update">Submit</button>',
```

Comment:
@squallstar Is this supposed to always be there on the submit button or only when it is an update?